test/unit/install/install_test.js

```diff
@@ -10,12 +10,60 @@
  */
 'use strict';
 
+const path = require('path');
+const sinon = require('sinon');
 const assert = require('chai').assert;
+const ProjectConfig = require('polymer-project-config').ProjectConfig;
+const PolymerCli = require('../../../lib/polymer-cli').PolymerCli;
 
 const polymerInstall = require('../../../lib/install/install');
 
 suite('install', () => {
 
+  suite('runs command', () => {
```

Comment:
This is testing the install command, which is different from the install logic run by the command.

Move these to a new, separate file in `tests/unit/commands/install_test.js`. Look at `commands/help_test.js` as an example.